pallets/identity/src/lib.rs

```diff
@@ -2138,64 +2137,57 @@ impl<T: Config> CheckAccountCallPermissions<T::AccountId> for Module<T> {
         pallet_name: &PalletName,
         function_name: &DispatchableName,
     ) -> Option<AccountCallPermissionsData<T::AccountId>> {
-        // `who` is the original origin (signer) of the original extrinsic
-        // context::current_identity is the target did against which the calling key must have permissions
-        // if `who`'s identity does not match context::current_identity we have a forwarded call. In this case the
-        // key for which we check permissions is `who`'s identity rather than `who`. This assumes that recursive forwarded calls
-        // are not allowed (which are prevented in `forwarded_call`)
-
-        if !<KeyToIdentityIds<T>>::contains_key(who) {
-            // Caller has no Identity
-            return None;
-        }
+        // `who` is the original origin (signer) of the original extrinsic.
+        // `context::current_identity` (CI) is the target DID for which the calling key must have permissions.
+        // If `who`'s identity does not match CI we have a forwarded call.
+        // In this case, the key for which we check permissions is `who`'s DID rather than `who`.
+        // This assumes that recursive forwarded calls, are not allowed, as enforced in `forwarded_call`.
```

Comment:
We don't have a `forwarded_call`.  I think this comment is referring to some older logic?

The only reason I see `who`'s identity not matching `context::current_identity` is from `CddAndFeeDetails` when a different identity pays the transaction fee (for accepting an authorization).

Reply:
Also we have a `ForwardedCall` event that is not created anywhere.

Reply:
forwarded calls were removed in #990